source/gems/dry-types/built-in-types.html.md

```diff
@@ -61,18 +61,18 @@ Assuming you included types in a module called `Types`:
   - `Types::Coercible::Hash`
 
 * Types suitable for `Form` param processing with coercions:
```

Comment:
Maybe now it'll be clearer to switch the backquotes to say "Types suitable for form \`Param\` processing with coercions".